keras_cv/models/object_detection/faster_rcnn/faster_rcnn.py

```diff
@@ -459,41 +349,48 @@ def compile(
         }
         super().compile(loss=losses, **kwargs)
 
-    def compute_loss(self, images, boxes, classes, training):
+    def compute_loss(self, x, y, y_pred, sample_weight, **kwargs):
```

Comment:
@ariG23498, you would need to mask out the `sample_weights` for box labels -1(this would ignore loss tensors for bboxes outside of image and also the padded bboxes you would get form using bounding_boxes.to_dense) . Please refer to [retinanet example here](https://github.com/keras-team/keras-cv/blob/e360fb7047f4c628359c9b5555e313a75c0f9a77/keras_cv/models/object_detection/retinanet/retinanet.py#L457)

Reply:
Addressed this in https://github.com/keras-team/keras-cv/pull/2012/commits/134f897651b1387db39212ae3d56ac90b95d0632

Reply:
Thank you!